README.md

```diff
@@ -423,13 +423,59 @@ for (const item of identities.items) {
 | `items` | [IEveryAuthIdentity](#ieveryauthidentity)[] | An array of EveryAuth credential objects. |
 | `next` | string \| undefined | If present, indicates that additional results may be obtained by supplying the `next` parameter to another call to `getIdentities` |
 
+#### deleteIdentity(serviceId, identityId)
+
+Deletes an existing identity.
+
+```javascript
+import everyauth from "@fusebit/everyauth-express";
+
+// Delete single identity
+const identityId = "idn-123"; // returned as part of getIdentitity and getIdentities
```

Comment:
Make it a real example
```
const identity = await everyauth.getIdentity('slack', { userId });
await everyauth.deleteIdentity('slack', identity.fusebit.identityId);
```

Also, side note: single quotes in code samples, not double quotes, here and elsewhere.